### ISSUE TITLE
error train result when using pytorch1.0.0

### ISSUE THREAD

**Issue by zhuixunforever** [BingCS/AtLoc#4]
I train your code with pytorch1.0.0, cuda 10, python2.7, no display error, and can be successfully trained, but val loss does not decrease, because my GPU is RTX2080, need to install cuda10, and I can't use pytorch 0.4.1
